src/views/Workflow.vue

```diff
@@ -136,9 +161,9 @@ export default {
     this.isLoading = true
     // clear the tree with current workflow data
     this.tree.clear()
+    this.table.clear()
```

Comment:
`this.table` does not have the `.clear()` function, since `this.table` is an [Array](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array).

Probably this can be used instead: `this.table.length = 0` (empties an array in JS... or `this.table = []`, thought would need to confirm Vue keeps reactivity that way too)

Reply:
Line replaced with `this.table = []` and table appears to be functioning fine following the change.